tests/integration/install/test_installation.py

```diff
@@ -467,3 +467,16 @@ def test_installation_with_dependency_upload(ws, installation_ctx, mocker):
 
     installation_ctx.deployed_workflows.repair_run("failing")
     assert installation_ctx.deployed_workflows.validate_step("failing")
+
+
+def test_account_installer_returns_workspace_contexts(env_or_skip, installation_ctx):
```

Comment:
This integration test fails. Though, we need the workspace contexts for proper integration testing. As mentioned in the PR top comment, I think we should make the integration testing part of #2507